src/app/content/components/Sidebar/styled/index.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { ComponentType } from 'react';
+import styled, { css } from 'styled-components/macro';
+import { CaretDown } from 'styled-icons/fa-solid/CaretDown/CaretDown';
+import { CaretRight } from 'styled-icons/fa-solid/CaretRight';
+import { textStyle } from '../../../../components/Typography';
+import theme from '../../../../theme';
+import { ArchiveTree } from '../../../types';
+import { splitTitleParts } from '../../../utils/archiveTreeUtils';
+import ContentLinkComponent from '../../ContentLink';
+
+export * from './wrapper';
+
+const numberCharacterWidth = 0.8;
+const numberPeriodWidth = 0.2;
+const iconSize = 1.7;
+const tocLinkHover = css`
+  :hover {
+    color: ${theme.color.text.black};
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+export const SummaryTitle = styled.span`
+  ${tocLinkHover}
+  font-size: 1.4rem;
+  line-height: 1.6rem;
+  font-weight: normal;
+  width: 100%;
+  display: flex;
+`;
+
+// tslint:disable-next-line:variable-name
+export const ContentLink = styled(ContentLinkComponent)`
+  ${textStyle}
+  ${tocLinkHover}
+  display: flex;
+  margin-left: ${iconSize}rem;
+  text-decoration: none;
+
+  li[aria-label="Current Page"] & {
+    font-weight: bold;
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+export const NavItemComponent: ComponentType<{active?: boolean, className?: string}> = React.forwardRef(
+  ({active, className, children}, ref) => <li
+    ref={ref}
+    className={className}
+    {...(active ? {'aria-label': 'Current Page'} : {})}
+  >{children}</li>
+);
+
+// tslint:disable-next-line:variable-name
+export const NavItem = styled(NavItemComponent)`
+  ${textStyle}
+  list-style: none;
+  font-size: 1.4rem;
```

Comment:
this font stuff should be on `ContentLink`, and should be shared with the font stuff on `SummaryTitle`